keyboards/hhkb/keymaps/dbroqua/keymap.c

```diff
@@ -3,19 +3,9 @@
  */
 #include QMK_KEYBOARD_H
 
-enum planck_layers
-{
-    _DEFAULT,
-    _ALTERNATE,
-    _FN,
-    _ADJUST
-};
-
-enum planck_keycodes
-{
-    DEF = SAFE_RANGE,
-    ALT
-};
+#define _DEFAULT 0
+#define _ALTERNATE 1
+#define _FN 2
```

Comment:
Is there a reason you didn't just stick with the `enum` construct above?

Reply:
Hi,

No particular reason, just an old habit.

Reply:
For a low number of layers like this, it's not really a dealbreaker IMO, we just generally prefer the `enum` convention.

Reply:
Ok, it's changed.